src/test/java/com/marklogic/mule/extension/AbstractFlowTester.java

```diff
@@ -61,14 +66,20 @@ int runFlowForDocumentCount(String flowName) {
 
     private DocumentData toDocumentData(Message message) {
         String content;
+        DocumentAttributes attributes;
         try {
             content = getPayloadAsString(message);
+            String attributesJson = new BufferedReader(
```

Comment:
Use Spring's FileCopyUtils here - Mule allows it, and it makes most of the code go away. Should be able to do:

```
String json = new String(FileCopyUtils.copyToByteArray(inputStream));
```